### PR TITLE
truffle-contract abstraction defaults

### LINKED ISSUE #55: invalid address during registry.apply
`this.registry` is a `truffle-contract` abstraction, which should send transactions from the `window.web3.eth.defaultAccount` address (no need to specify {from: this.account})

This line is throwing an `invalid address` error
https://github.com/AdChain/AdChainRegistryWebApp/blob/eb6c8d71886244e04b4829e444070400aa0b9818/src/services/registry.js#L97

If I put back {from: this.account}, it works. 

truffle-contract bug?

### PR DESCRIPTION
**Pull request by kangarang**
- Sets default {from: account} in truffle-contract abstractions
- Fixes #55

### CODE REVIEW

File: src/services/parameterizer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ class ParameterizerService {
   }
 
   async init () {
-    this.parameterizer = await getParameterizer()
+    this.parameterizer = await getParameterizer(window.web3.eth.defaultAccount)
```
Comment (miguelmota) [inline src/services/parameterizer.js]: @kangarang `window.web3` is undefined if MetaMask isn't installed and will throw error here. Can't assume window.web3 is available

**Comment by miguelmota** [comment]
nice, didn't know about `defaults`

File: src/services/parameterizer.js (hunk 1 above)
Comment (kangarang) [reply src/services/parameterizer.js]: @miguelmota fasho. Lemme think about this this afternoon.

File: src/services/token.js [hunk 2]
```diff
@@ -11,7 +11,12 @@ class TokenService {
   }
 
   async init () {
-    this.token = await getToken()
+    this.token = await getToken(window.web3.eth.defaultAccount)
+    if (!this.token) {
+      return false
```
Comment (miguelmota) [inline src/services/token.js]: this check shouldn't be necessary, since init is only called at one place on load

**Pull request merged**